engine/swarm/services.md

```diff
@@ -288,7 +288,17 @@ $ docker service create \
 Service constraints let you set criteria for a node to meet before the scheduler
 deploys a service to the node. You can apply constraints to the
 service based upon node attributes and metadata or engine metadata. For more
-information on constraints, refer to the `docker service create` [CLI  reference](../reference/commandline/service_create.md).
+information on constraints, refer to the `docker service create` [CLI reference](../reference/commandline/service_create.md).
+
+Placement preferences let you divide tasks evenly over different categories of
+nodes. An example of where this may be useful is balancing tasks between
+multiple datacenters or availability zones. Using a placement preference, you
+can avoid situations where too many of service's tasks are scheduled on nodes
+in a particular datacenter, making the service vulnerable to a localized
+outage. Additional placement preferences can be used to further divide tasks
```

Comment:
You can use additional placement preferences to further divide tasks over groups of nodes. For example, you can balance them....

Try to avoid semicolons and long sentences, for ease of use by non-native English readers.